server/services/placementRequestService.ts

```diff
@@ -7,7 +8,18 @@ export default class TaskService {
   async getAll(token: string) {
     const placementRequestClient = this.placementRequestClientFactory(token)
 
-    const tasks = await placementRequestClient.all()
-    return tasks
+    const results = {
+      not_matched: [],
+      unable_to_match: [],
+      matched: [],
+    } as GroupedPlacementRequests
+
+    const placementRequests = await placementRequestClient.all()
+
+    placementRequests.forEach(placementRequest => {
+      results[placementRequest.status].push(placementRequest)
```

Comment:
I think it would be better to be consistent with camelCasing throughout the codebase.